<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShellCalculator</class>
 <widget class="QMainWindow" name="ShellCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShellCalculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #ff3d00;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(0, 0, 0);
font: 16pt &quot;Calibri&quot;;
color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="b_parentheses">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_delete">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_percent">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_nine">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_eight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_seven">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_multiply">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_plus">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>550</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_five">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_four">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_six">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_three">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>550</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_two">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_minus">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_one">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_plus_or_minus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_zero">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>640</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_equals">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>640</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #00c6ff;
color : black;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_point">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>640</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : white;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_square_root">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_pow">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>x^y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_pow_two">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>x²</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_division">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
color : #00c6ff;
border-radius : 40px;
font: 20pt &quot;Source Code Pro&quot;;</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #333333;
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
